docs/meshstack.aws.metering.md

```diff
@@ -70,15 +70,18 @@ In order for meshStack to generate Usage Reports, following policies are require
 
 ## Reserved Instances & Savings Plans
 
-meshStack collects the information about reserved instances and savings plans via the cost explorer API.
+This section applies only if your meshCustomers buy Reserved Instances and Savings Plans which give priority to their
+AWS tenants. If this is the case you can enable `reservedInstanceFairChargeback` and `savingsPlanFairChargeback` feature
```

Comment:
AWS tenants is ambigious -> meshTenant? AWS Account? 

We should also mention that the RI/SP is bought directly on the account, so that the account receives priority for consuming the benefits of the RI/SP.

Reply:
Done.